R/pkg/R/sparkR.R

```diff
@@ -394,6 +394,16 @@ sparkR.session <- function(
                                 sparkConfigMap,
                                 enableHiveSupport)
     assign(".sparkRsession", sparkSession, envir = .sparkREnv)
+
+    # Set maximum size of serialized data that can be sent as invokeJava arguments
+    assign(
+      ".maxArgByteSize",
+      callJMethod(
+        callJMethod(sparkSession, "conf"),
+        "get",
+        "spark.r.maxArgByteSize",
+        toString(.Machine$integer.max * 10)),
+      envir = .sparkREnv)
```

Comment:
also, you might want to call `sparkR.conf("spark.r.maxArgByteSize",  toString(.Machine$integer.max * 10))`